README.md

```diff
@@ -1,140 +1,141 @@
+7. Start the front-end workflow. After that, our frontend prototype should be accessible via `http://localhost:3000/demo/`:
 
         $ gulp
 
-Hat nicht geklappt? Sorry :-(  
-Nicht ärgern und lieber im [REDAXO-Slack](http://redaxo.org/slack/) nachfragen oder im [Forum](http://www.redaxo.org/de/forum/allgemeines-f39/frontend-workflow-fur-redaxo-mit-gulp-browserify-postcss-t21541.html#p120663), dort wird dir geholfen!
+Did not work? Sorry :-(  
+No worries! You can ask [REDAXO-Slack](http://redaxo.org/slack/) or in the [Forum](http://www.redaxo.org/de/forum/allgemeines-f39/frontend-workflow-fur-redaxo-mit-gulp-browserify-postcss-t21541.html#p120663), here you will get help!
 
 ---
 
 ![Screenshot](https://raw.githubusercontent.com/FriendsOfREDAXO/redaxo-mit-bimmelbam/assets/redaxo-mit-bimmelbam_02.png)
 
-## Wie geht’s weiter?
+## What's next?
 
 ### REDAXO
 
-In diesem Projekt ist _keine_ REDAXO-Installation enthalten. Wenn deine Gulp-Tasks erfolgreich durchgelaufen sind, kannst du dir REDAXO nun selbst dazustecken, und zwar direkt ins `/app`-Verzeichnis. Dort liegen bereits die typischen REDAXO-Ordner.
+This project does not contain _no_ REDAXO installation. Once your gulp tasks have completed successfully, you can now put REDAXO yourself, directly into the `/app` directory. There are already the typical REDAXO folders.
 
-Sobald dein REDAXO läuft, kannst du die Website wie gewohnt mittels Templates und Modulen implementieren. Das HTML verwendest du so wie im Prototyp (ggfls. Pfade anpassen!) und bindest die gleichen Assets ein. Am Ende hast du eine lauffähige REDAXO-Website, deren Output mit dem Prototyp identisch ist.
+As soon as your REDAXO is up and running, you can implement the website as usual using templates and modules. The HTML you use as in the prototype (if necessary, adapt paths!) And embed the same assets. In the end, you have a working REDAXO website whose output is identical to the prototype.
 
-### Prototyp
+### Prototype
 
-Falls du dein Frontend zukünftig direkt in REDAXO entwickeln möchtest, kannst du den Prototyp nun verwerfen. __Jedoch ist die Absicht dieses Projekts ausgerechnet die, dich davon zu überzeugen, Frontend grundsätzlich _außerhalb_ von REDAXO anhand des Prototyps zu entwickeln.__ Warum? Weil es effizienter und komfortabler ist, und weil es im Team ggfls. von Leuten übernommen werden kann, die sich ausschließlich auf Frontend-Entwicklung konzentrieren und keine REDAXO- und PHP-Erfahrung benötigen.
+If you want to develop your frontend directly in REDAXO in the future, you can now discard the prototype. __But the purpose of this project is to convince you to develop the frontend basically _outside_ of REDAXO based on the prototype .__ Why? Because it is more efficient and comfortable, and because it works in a team. can be adopted by people who focus exclusively on frontend development and do not need a REDAXO and PHP experience.
```

Comment:
Proposal: »If you prefer to develop your frontend within REDAXO from now on, you can discard the prototype. `__`However, the intention of this project is to convince you to develop frontend `_`outside`_` of REDAXO using the prototype.`__` Why? Because it’s more efficient and comfortable, and also because it can be done by fellow workers who concentrate exclusively on frontend development and don’t require REDAXO and PHP experience.«